From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 30 Jun 2008 14:09:39 -0400
Subject: [misc] fix UP compile in skcipher.h
Message-id: 20080630180814.8030.72243.sendpatchset@prarit.bos.redhat.com
O-Subject: [RHEL5 PATCH]: Fix UP compile in skcipher.h
Bugzilla: 453038
RH-Acked-by: Rik van Riel <riel@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

Fix UP (CONFIG_SMP is not set) compile.

Add missing header file to skcipher.h.

Successfully tested by me.

Resolves BZ 453038.

 include/crypto/internal/skcipher.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/crypto/internal/skcipher.h b/include/crypto/internal/skcipher.h
index c970f20..0fd6234 100644
--- a/include/crypto/internal/skcipher.h
+++ b/include/crypto/internal/skcipher.h
@@ -16,6 +16,7 @@
 #include <crypto/algapi.h>
 #include <crypto/skcipher.h>
 #include <linux/types.h>
+#include <linux/interrupt.h>
 
 struct rtattr;
 
